### ISSUE TITLE
[Feature request] Show only platforms that have the requirements of the filter

### ISSUE THREAD

**Issue by BanCrash** [felixkmh/DuplicateHider#37]
Hello! I don't know if this is tecnichally possible, but I would like that when you use a filter, it shows only the platforms that accomplish the requirements of the filter.

For example, I have Apex Legends installed on Steam but not on Origin, and when I use the Installed filter, it shows both platforms on top right, when IMO it should show only the platform that have the requirements of the filter (steam in this case):

![image](https://user-images.githubusercontent.com/11770760/140927903-1ce99ce2-715f-4095-bc11-838abb5c77f5.png)

Steam:
![image](https://user-images.githubusercontent.com/11770760/140927943-732af919-a194-49b1-8aed-33637f7aa4e4.png)
Origin:
![image](https://user-images.githubusercontent.com/11770760/140927991-3b23327a-2a1d-4462-b7b6-c38d1dfb689b.png)

Like I've said, I don't know if this is technically possible.

**Comment by felixkmh** [comment]
I don't think that something like that is currently supported by the API exposed by Playnite. It might be doable once https://github.com/JosefNemec/Playnite/issues/2600 is implemented.

**Comment by BanCrash** [comment]
Just for tracking purpose: https://github.com/JosefNemec/Playnite/commit/aa93a0c7c19d9ffa0991d1044dbe670559a86aa5

**Comment by BanCrash** [comment]
Also maybe this https://github.com/JosefNemec/Playnite/commit/3ea431d893825de5babfa3d7414f0f8d064ee8a4

Maybe with this and the previous comment commits is this already doable?
